ivy/functional/frontends/paddle/nn/functional/pooling.py

```diff
@@ -24,3 +25,37 @@ def avg_pool1d(
         ceil_mode=ceil_mode,
         data_format=data_format,
     )
+
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+def avg_pool3d(
+    x,
+    kernel_size,
+    stride=(1, 1, 1),
+    padding=(0, 0, 0),
+    /,
+    *,
+    data_format="NDHWC",
+    count_include_pad=False,
+    ceil_mode=False,
+    divisor_override=None,
+    out=None,
+):
```

Comment:
`exclusive` arg is missing from the implementation

Reply:
```exclusive``` is actually not present in any backend file, to include this arg entire backend needs to be modified